### DISCUSSION TITLE
umi里系统名称的国际化问题咨询

### DISCUSSION THREAD

**Discussion by joraboy328** [umijs/umi#11506] [General]
在umi 配置文件layout配置里 name或title属性为应用顶部现实的系统名称，在i18n里如何配置？
en-US.js和zh-CN.js我已配置，并且启用了i18n功能。

  layout:{
    name:'应用系统名称',
    contentWidth: 'Fluid',
    splitMenus: false,
    fixedHeader: false,
.....

  },

**Comment by fz6m** [comment]
在 `src/app.tsx` 运行时里配，参考 [`export const layout`](https://umijs.org/docs/max/layout-menu#%E8%BF%90%E8%A1%8C%E6%97%B6%E9%85%8D%E7%BD%AE) 。

**Reply by joraboy328** [reply]
谢谢，我目前用的是umi3，咋实现呢？

**Reply by fz6m** [reply]
umi 3 不维护了，建议升级到 umi 4

**Reply by joraboy328** [reply]
好的，谢谢！
不过现在项目是用umi3的，暂时还升级不了，能否先提供一下umi3下此问题解决方法？
多谢！
